src/handbook/company/security/access-control.md

```diff
@@ -0,0 +1,108 @@
+---
+navTitle: Access Control Policy
+---
+
+# Access Control Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @knolleary     | 2023-05-01     |
+
+## Purpose
+
+To limit access to information and systems to authorized parties in accordance
+with business objectives.
+
+## Scope 
+
+This policy applies to all FlowForge information systems that process, store, or
+transmit confidential data as defined in the FlowForge [Data Management Policy](./data-management.md).
+
+It applies to all employees of FlowForge and to all external parties with access
+to FlowForge system resources.
+
+## Access Control Policy
+
+1. Access to all computing resources, including servers, end-user computing
+  devices, network equipment, services and applications, must be protected by
+  strong authentication, authorization, and auditing.
+
+2. Interactive user access must be associated to an account or login unique to
+each user.
+
+3. All credentials, including user passwords, service accounts, and access
+keys, must meet the length, complexity, age, and rotation requirements defined
+in security standards.
+
+4. Use strong password and multi-factor authentication (MFA) whenever possible
+to authenticate to all computing resources.
+
+5. MFA is required to access any critical system or resource, including but not
+limited to resources in production environments.
+
+6. Unused accounts, passwords, access keys must be removed within 30 days
+
+7. A unique access key or service account must be used for different
+application or user access.
+
+8. Authenticated sessions must time out after a defined period of inactivity.
+
+
+### Access Authorization and Termination
+
+1. Access authorization shall be implemented using role-based access control
+(RBAC) or similar mechanism.
+
+2. Standard access based on a user's job role may be pre-provisioned during
+employee onboarding. All subsequent access requests to computing resources must
+be approved by the requestor’s manager, prior to granting and provisioning of
+access.
+
+3. Access to critical resources, such as production environments, must be
+approved by the CTO in addition to the requestor’s manager.
+
+4. Access must be reviewed on a regular basis and revoked if no longer needed.
+
+5. Upon termination of employment, all system access must be revoked and user
```

Comment:
Should we start an off boarding template? So when we need it, this policy is actually effective? cc @Tracy-Anthony

Reply:
We will eventually have a `Human Resource Security Policy` that I believe will cover this and we can add cross-links once it's in place.

Reply:
Yes, we should. I started working on the initial bit of the policy already.